outbox/register-postgres.json

```diff
@@ -13,6 +13,7 @@
     "transforms" : "outbox",
     "transforms.outbox.type" : "io.debezium.transforms.outbox.EventRouter",
     "transforms.outbox.route.topic.replacement" : "${routedByValue}.events",
-    "transforms.outbox.table.field.event.timestamp" : "timestamp",
-    "transforms.outbox.table.fields.additional.placement" : "type:header:eventType"
+//    "transforms.outbox.table.field.event.timestamp" : "timestamp", TODO INT64 exception
```

Comment:
Please remove this line, comments will result into an error. I don't think this `timestamp` is actually needed.